File: BookShop.BusinessLogic/Models/PrintedEditions/PrintedEditionPageModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace BookShop.BusinessLogic.Models
{
    public class PrintedEditionPageModel
    {
        public int CurrentPage { get; set; }
        public int PageSize { get; set; }
        public int Count { get; set; }
        public IEnumerable<PrintedEditionModel> PrintedEditionModels { get; set; }
    }
}

File: BookShop.ClientApp/src/app/models/printedEditionPageModel.ts
import { PrintedEditionModel } from 'src/app/models/printedEditionModels/printedEditionModel';

export class PrintedEditionPageModel {
    public currentPage: number;
    public pageSize: number;
    public count: number;
    public printedEditionModels: PrintedEditionModel[];
    constructor() {

    }
}